src/borg/archiver.py

```diff
@@ -1326,7 +1326,7 @@ def do_prune(self, args, repository, manifest, key):
             keep += prune_split(archives, '%Y', args.yearly, keep)
         to_delete = (set(archives) | checkpoints) - (set(keep) | set(keep_checkpoints))
         stats = Statistics()
-        with Cache(repository, key, manifest, do_files=args.cache_files, lock_wait=self.lock_wait) as cache:
+        with Cache(repository, key, manifest, do_files=False, lock_wait=self.lock_wait) as cache:
```

Comment:
for prune, we only use the chunks cache (as expected), so do_files can be just always False.